java-frontend/src/main/java/org/sonar/plugins/java/api/JavaFileScannerContext.java

```diff
@@ -75,7 +76,9 @@
 
   void reportIssue(JavaCheck javaCheck, Tree tree, String message, List<Location> secondaryLocations, @Nullable Integer cost);
 
-  void reportIssue(JavaCheck javaCheck, Tree starTree, Tree endTree, String message);
+  void reportIssue(JavaCheck javaCheck, Tree startTree, Tree endTree, String message);
+
+  void reportIssue(JavaCheck javaCheck, Tree startTree, Tree endTree, String message, List<Location> secondaryLocations);
```

Comment:
Ok, adding `@Nullable` cost to match other API. First method already exists, I just fixed a typo.